editor/scripts/bundle.py

```diff
@@ -153,19 +153,31 @@ def sign_files(platform, options, dir):
         return
     if 'win32' in platform:
         certificate = options.windows_cert
-        certificate_pass = options.windows_cert_pass
+        certificate_pass_path = options.windows_cert_pass
         if certificate == None:
             print("No codesigning certificate specified")
             sys.exit(1)
 
+        if not os.path.exists(certificate):
+            print("Certificate file does not exist:", certificate)
+            sys.exit(1)
+
+        certificate_pass = 'invalid'
+        with open(certificate_pass_path, 'rb') as f:
+            certificate_pass = f.read()
+
+        signtool = os.path.join(os.environ['DYNAMO_HOME'], 'ext','SDKs','Win32','WindowsKits','10','bin','10.0.18362.0','x64','signtool.exe')
```

Comment:
Not sure how to find the tool on the system, so I used our packaged sdk instead.

Reply:
I assumed it was installed and on the path on GitHub CI Windows machines?